pyproject.toml

```diff
@@ -93,7 +93,7 @@ requests = "^2.23"
 matplotlib = ">=3.1,<3.6"
 attrs = ">=19.3,<22.2"
 jsonpickle = ">=1.3,<3.1"
-redis = ">=3.4,<5.0"
+redis = "4.3.6 || 4.4.3 || 4.5.3"
```

Comment:
Could we actually have this on 3.5.x to:
```suggestion
redis = ">=4.5.3, <5.0"
```
This way we can receive dependabot updates for future upgrades.

Reply:
If the vulnerability is flagged on all supported versions, this will have to change accordingly per release branch:
3.3.x -> ">=4.3.6, <5.0"
3.4.x -> ">=4.4.3, <5.0"

Reply:
Oh I see, good shout!
I'd either pin the dependency directly (e.g. 3.3.x -> "4.3.6"), or just add the same constraint as in `3.5.x`.
Not sure how poetry interprets the "||" 🤔 I haven't seen this usage in their docs, but possibly I missed it too.